src/main/scala/no/ndla/audioapi/integration/MigrationApiClient.scala

```diff
@@ -8,23 +8,25 @@
 
 package no.ndla.audioapi.integration
 
-import no.ndla.audioapi.AudioApiProperties
+import no.ndla.audioapi.AudioApiProperties.{MigrationHost, MigrationPassword, MigrationUser, Environment}
 import no.ndla.network.NdlaClient
 
 import scala.util.Try
 import scalaj.http.Http
+import com.netaporter.uri.dsl._
 
 trait MigrationApiClient {
   this: NdlaClient =>
   val migrationApiClient: MigrationApiClient
 
   class MigrationApiClient {
-    val audioMetadataEndpoint = s"${AudioApiProperties.MigrationHost}/audio/:audio_id"
+    val DBSource = if (Environment == "prod") "cm" else "red"
```

Comment:
Burde vi legge inn muligheten for en override fra kommandolinje?
Denne blir for enkel for "det nye miljøet", men vi kan jo fikse det når det faktisk skjer også selvfølgelig...

Reply:
Hvis vi åpner for å override via kommandolinje åpner vi jo for at man potensielt kan få data fra blandede kilder i samme miljø. Det er vel noe vi øsnker å unngå? Kan selvfølgelig legge inn i environmentet hvilke miljøer som skal importere fra cm og hvilke som skal importere fra red. Usikker på hvilken måte jeg liker best

Reply:
Det er et godt poeng. Vi lar det være som det er nå. Så kan vi heller se på alternativer hvis det blir et praktisk behov senere...